src/makeRobot.js

```diff
@@ -39,6 +39,66 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
якщо назви однакові то їх можна не прописувати двічі
```suggestion
    name,
    wheels,
    version,
```